tests/unit/pipeline.spec.ts

```diff
@@ -159,7 +160,7 @@ describe('Pipeline.vue', () => {
       ];
       const store = setupMockStore(buildStateWithOnePipeline(pipeline));
       dispatchSpy = jest.spyOn(store, 'dispatch');
-      wrapper = mount(PipelineComponent, { store, localVue });
+      wrapper = mount(PipelineComponent, { store, localVue, stubs: { FAIcon: '<div />' } });
```

Comment:
Tiny detail: maybe it's best to stub this component with an empty <svg/> as it'll more close to the actual result

Reply:
Also, to avoid having to mock FAIcon repeatidly, you can write a stub component in a `__mocks__` folder just beside FAIcon.vue, and then you'll only have to write once `jest.mock('@/components/FAIcon.vue')` per test file.
And it may also ease following tests writing :)

Reply:
Je l'ai dev ici https://github.com/ToucanToco/weaverbird/pull/1047/commits/d3ba6f1f6783ec2a5e86dc1e6cfdd8446842c9ad, par contre j'ai du l'importer dans la quasi totalité des fichier (mais ça clean toutes les console error qui trainaient surement, c'est pas plus mal. Pas besoin de tout review t'as plus de 60 fichiers d'impactées :sweat_smile: )